packages/rancher-logging/4.10/generated-changes/patch/templates/_helpers.tpl.patch

```diff
@@ -97,17 +97,18 @@
 +    Regex             ^(?<level>[IWEF])(?<timestamp>\d{4} \d{2}:\d{2}:\d{2}).\d{6} +?(?<thread_id>\d+) (?<filename>.+):(?<linenumber>\d+)] (?<message>.+)
 +    Time_Key          timestamp
 +    Time_Format       %m%d %T
++    Time_Keep         On
 +
 +[PARSER]
 +    Name              rancher
 +    Format            regex
 +    Regex             ^time="(?<timestamp>.+)" level=(?<level>.+) msg="(?<msg>.+)"$
 +    Time_Key          timestamp
-+    Time_Format       %FT%H:%M:%S
++    Time_Format       %FT%H:%M:%S:::z
```

Comment:
I don't understand what is going on here. According to https://docs.fluentbit.io/manual/pipeline/parsers/configuring-parser, [strptime(3)](https://linux.die.net/man/3/strptime) is used. `strptime(3)` would parse the timezone using `%z`, so wouldn't we want `Time_Format` to be set to `%FT%H:%M:%S%z`, if the time is in the format `2025-05-22T16:40:20+02:00` as is written in https://github.com/rancher/ob-team-charts/issues/84?

But wait, the timezone part of that is `+02:00`, not `+0200`, which means it doesn't conform to RFC 822... I'm confused. Can you please explain your reasoning here?

Reply:
Actually, it seems that `%z` does support ISO 8601 in most modern Linux distributions. I tested this on Ubuntu and got this
<img width="526" alt="image" src="https://github.com/user-attachments/assets/ea1a902f-9491-491a-8f78-c76f522cbc44" />

I will update the pr to use it